browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2993,7 +2993,10 @@ If you don't accept this request, VPN will not reconnect and your internet conne
       Disconnect
     </message>
     <message name="IDS_FRAGMENT_SIGN_MESSAGE_DESCRIPTION" desc="Sign message dialog description.">
-      Your signature is being requested
+      A DApp is requesting your public encryption key
+    </message>
+    <message name="IDS_BRAVE_WALLET_PROVIDE_ENCRYPTION_KEY_DESCRIPTION" desc="Provide encryption key panel details">
+      is requesting your wallets public encryption key. If you consent to providing this key, the site will be able to compose encrypted messages to you.
```

Comment:
wallet's?

Reply:
yes @qamarngr could you pls fix and uplift?

Reply:
Same in `IDS_BRAVE_WALLET_PROVIDE_ENCRYPTION_KEY_TITLE` in `components/resources/wallet_strings.grdp`

Reply:
@bbondy will fix it this

Reply:
@mkarolin What should be fixed in `IDS_BRAVE_WALLET_PROVIDE_ENCRYPTION_KEY_TITLE`

Reply:
@qamarngr it should be "wallet's" as well in `wallet_strings.grdp` file (desktop string file)

Reply:
https://ci.brave.com/job/brave-core-create-uplift-prs/1917/